docs/core/tools/csproj.md

```diff
@@ -271,7 +271,7 @@ Here is the complete list of [SPDX license identifiers](https://spdx.org/license
 
 The exact syntax of the license expressions is described below in [ABNF](https://tools.ietf.org/html/rfc5234).
 
-```cli
+```console
```

Comment:
Not sure if we have a good identifier for this one. Maybe leave empty in this case?